### DISCUSSION TITLE
NoClassDefFoundError URIBuilder using MSAL 4.2.0 for Android

### DISCUSSION THREAD

**Discussion by alexandru-spasenie** [AzureAD/microsoft-authentication-library-for-android#1797] [Q&A]
Hello, I get a crash when trying to create the `PublicClientApplication` object caused by `java.lang.ClassNotFoundException: Didn't find class "cz.msebera.android.httpclient.client.utils.URIBuilder"`
I tried with the official MSAL sample Android project and it seems to be working fine. I suspect it's a library conflict problem that's causing the crash. Maybe I have to exclude a module from the MSAL library or add another library that's needed. I tried to exclude `org.apache.httpcomponents` but no use. 
![image](https://user-images.githubusercontent.com/14100133/230932321-87af2905-fdd7-4139-b781-a8ede68ee063.png)
![image](https://user-images.githubusercontent.com/14100133/230931390-6539fbb6-4395-4270-89a0-d91e4750d725.png)
![image](https://user-images.githubusercontent.com/14100133/230931545-9de7e95c-21ad-4779-b0ed-11feeca00178.png)

Any help or tips are greatly appreciated!

**Comment by negoe** [comment]
@alexandru-spasenie Please create an issue if you are still experiencing this problem. We do not monitor this channel actively for customer issues. For any GitHub issues we recommend posting your questions [here](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues).
